text/0001-cdk-update.md

```diff
@@ -0,0 +1,285 @@
+ * LambdaFunction[StreamFunction]: <spinner>
+
+```
+
+The watcher can only watch for file changes. For Lambda code, that means that
+directories will be watched for changes and zipped if they change. If the file
+asset is a zip file, then the update will fire whenever that changes. The
+existing docker asset builder will be used to watch for changes in local docker
+images.
+
+#### Resource Support
+
+- AWS Lambda `Function`
+  - file and directory assets
+- AWS Fargate
+  - image assets
+- ECS
+  - image assets
+
+#### Future Support Plans
+
+- StepFunctions
+- API Gateway models
+
+### PRESS RELEASE
+
+DATE - AWS announces the `cdk deploy --accelerated` command for the AWS CDK
+toolkit. The `accelerate` command allows CDK application developers to rapidly
+update code within their CDK application when no other AWS resources are
+changing, bypassing the more correct, but time consuming, stack update
+procedure.
+
+The CDK toolkit uses CloudFormation under the hood to manage changes to AWS
+resources, which safely updates resources. This update process is slow, however,
+and during development it performs safety checks that add overhead to developer
+workflows. Those delays are interposed between every change a developer makes,
+and over the course of a product may add hours to the total development time.
+
+The `cdk deploy --accelerated` command provides a way to bypass those checks
+when working in development, directly updating the code in Lambda functions and
+ECS services without a full CDK deployment. When the `accelerate` command can
+identify that all changes in the stack have shortcut support, it will directly
+publish the assets to AWS and then modify the underlying CDK resources directly.
+
+> Using CDK accelerate dramatically improves our development iteration speed for
+> Lambda functions. It cuts down the change-deploy-test cycle to seconds, where
+> it was minutes before with a full cdk deploy each time. Unlike some other
+> solutions, it actually updates our function running on AWS, which means there
+> are no problems with the Lambda behaving differently during testing, and then
+> in production - Adam Ruka, developer
+
+Any CDK users who are deploying code to Lambda functions or containers on ECS or
+Fargate can take advantage of this tool today, by configuring their stacks for
+interactive update, and then using `cdk deploy --accelerated` instead of
+`cdk deploy`.
+
+## FAQ
+
+### What are we launching today?
+
+The `cdk deploy --accelerated` and `cdk deploy --accelerated --watch` commands,
+with support for rapid update of Lambda function code and Fargate images.
+
+### Why should I use this feature?
+
+If you are developing a CDK application and want to publish your code changes
+without waiting for CloudFormation to perform a full, safe stack update, you
+should use this feature.
+
+### Does it work when my Lambda is in Go or Java?
+
+To an extent, yes, but there are caveats. In order to update the lambda code you
```

Comment:
What about if the user is using the native bundling functionality (like with [aws-lambda-go](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-lambda-go-readme.html)? Will it be able to use the builtin asset bundler? Seems like it would be similar to how it would work for docker images.

Reply:
Good point, we should add a FAQ